source/userguide/entity_resolver.rst

```diff
@@ -171,51 +171,230 @@ Create the synonyms and make them available to your MindMeld app, going entity t
 
     e. Choose a new target entity (continue until you have covered all the entities for the entity type)
 
-#. Choose a new entity type (stop when you have covered all the entity types for the app) 
+#. Choose a new entity type (stop when you have covered all the entity types for the app)
 
+Access the entity resolver
+----------------------------
+
+Working with any natural language processor component falls into two broad phases:
+
+ - First, generate the training data for your app. App performance largely depends on having sufficient quantity and quality of training data.
+ - Then, conduct experimentation in the Python shell.
+
+Once all of the entity mapping files are generated, you can either (1) train the resolver as a standalone component, or (2) build the whole NLP pipeline, which trains the resolver along with the other components. We use the `Food Ordering blueprint <https://mindmeld.com/docs/blueprints/food_ordering.html>`_ to demonstrate using an entity resolver.
```

Comment:
By emphasizing the "training" phase here we seem to be giving the impression that the ER model requires some training, which is not the case. My suggestion is to remove the bullet points so we don't categorize this module as being similar to the others (atleast for now).

Reply:
Sure, will do that change.